uiowa/uiowa
# Feature: Sidebar blocks for AoS

Resolves https://github.com/uiowa/uiowa/issues/2966. 
Resolves https://github.com/uiowa/uiowa/issues/2964. 


# How to test

- `blt ds --site=admissions.uiowa.edu`
- `yarn workspace admissions_core build`
- `drush @admissions.local cr`
- Go to http://admissions.local.drupal.uiowa.edu/areas-of-study/accounting and verify that the related links and scholarship boxes are styled like the screenshots below:

<img width="1489" alt="Screen Shot 2021-02-03 at 2 04 29 PM" src="https://user-images.githubusercontent.com/1036433/106802722-f6300800-6628-11eb-8311-e6fd84bc0ba5.png">
<img width="1560" alt="Screen Shot 2021-02-03 at 2 04 17 PM" src="https://user-images.githubusercontent.com/1036433/106802732-f92af880-6628-11eb-987e-581daf85d9a8.png">
